### PR TITLE
[MD2C] Add command for the persistent grade

### PR DESCRIPTION
**Pull request by flying-pi**
Add new command for collecting historical persistent grades

### CODE REVIEW

File: lms/djangoapps/certificates/management/commands/generate_persistent_grade.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    def handle(self, *args, **options):
+        course_structure = None
+        for enrollment in CourseEnrollment.objects.all():
+            user = enrollment.user
+            try:
+                course = courses.get_course_by_id(enrollment.course_id)
+                course_structure = get_course_blocks(user, course.location)
+            except:
+                # Used for catch exceptions related to bad courses
```
Comment (idegtiarov) [inline lms/djangoapps/certificates/management/commands/generate_persistent_grade.py]: What exception are we expecting for?

File: lms/djangoapps/certificates/management/commands/generate_persistent_grade.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                pass
+            # Used for ignoring student that doesn't enrol in given course
```
Comment (idegtiarov) [inline lms/djangoapps/certificates/management/commands/generate_persistent_grade.py]: typo s/enrol/enroll/

File: lms/djangoapps/certificates/management/commands/generate_persistent_grade.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    def handle(self, *args, **options):
+        course_structure = None
+        for enrollment in CourseEnrollment.objects.all():
+            user = enrollment.user
+            try:
+                course = courses.get_course_by_id(enrollment.course_id)
+                course_structure = get_course_blocks(user, course.location)
+            except:
+                # Used for catch exceptions related to bad courses
+                pass
```
Comment (idegtiarov) [inline lms/djangoapps/certificates/management/commands/generate_persistent_grade.py]: I would expect some logs here

File: lms/djangoapps/certificates/management/commands/generate_persistent_grade.py (hunk 1 above)
Comment (flying-pi) [reply lms/djangoapps/certificates/management/commands/generate_persistent_grade.py]: The exception can occur when the student for some reason lost access to the course. In normal flow it is impossible, but in some cases, staff can restrict access to the course for the user. Or another case - course can be removed by sysadmin dashboard, but de facto it exists in the system.

**Pull request merged**